### ISSUE TITLE
Chrome - Video not visible to other participants in a meeting

### ISSUE THREAD

**Issue by venkatesh-jaam** [jitsi/jitsi-meet#7559]
### Description:

The video of user, browsing from Chrome is not visible to other participants. This issue is happening only when the user enters the meeting with video muted; and then unmutes the video later in the call.

We are facing this issue after we updated our application with this version of Jitsi Meet code [jitsi-meet_4917](https://github.com/jitsi/jitsi-meet/releases/tag/jitsi-meet_4917)

### Steps to reproduce:

1. Create a meeting in any browser
2. Copy the Invite link and open it in Chrome browser
3. Mute the Audio and Video before entering into the meeting
4. Once entered into the meeting, Unmute the Video

### Expected behavior:

The Umuted video of the user in Chrome browser should be visible to all other participants in the meeting.

### Actual behavior:

The video's unmute flag gets passed accross the participants, but the video is not even uploaded into websocket from chrome.
<img width="346" alt="chrome-short" src="https://user-images.githubusercontent.com/69686265/90796413-eba8dd00-e32c-11ea-871b-c409cf2ad4f7.png">
<img width="366" alt="firefox-short" src="https://user-images.githubusercontent.com/69686265/90796424-ef3c6400-e32c-11ea-95d7-2f6f1be7a9a0.png">

### Server information:

- Jitsi Meet version:  [jitsi-meet_4917](https://github.com/jitsi/jitsi-meet/releases/tag/jitsi-meet_4917)
- Operating System: macOS (10.15.6)

### Client information:

- Browser / app version: Chrome (Version 84.0.4147.135)
- Operating System: macOS (10.15.6)

### Additional information:

The websocket communication is good and there are no errors in the runtime. But the video stream alone is not being uploaded from Chrome's websocket (Referred from websocket payload information).

**Comment by damencho** [comment]
Do you reproduce this on meet.jit.si? What does local stats say is this p2p or going to the bridge. Next steps for debugging this is normally to check whether a stream is being received in chrome://webrtc-internals.

Please when you have questions or problems use the community forum before opening new issues, thanks.

**Comment by damencho** [comment]
You report client is chrome, but on the screenshot you wrote Firefox? 
Do you repro everytime, what if you use chrome on both sides, do you repro?

**Comment by venkatesh-jaam** [comment]
> You report client is chrome, but on the screenshot you wrote Firefox?

In firefox _(Refer screenshot **Application from Firefox browser**)_, the video from Chrome is not available. Because the video stream upload itself is not happening in Chrome.

Whereas in Chrome _(Refer screenshot **Application from Chrome browser**)_, the video from Firefox is received.

> Do you repro everytime, what if you use chrome on both sides, do you repro?

Yes I can but not consistently. I couldn't pin point to the exact error scenario.

**Comment by damencho** [comment]
> In firefox _(Refer screenshot **Application from Firefox browser**)_, the video from Chrome is not available. Because the video stream upload itself is not happening in Chrome.

How did you verify that?

What about the local stats of the chrome side when this happen? Is this p2p or bridge? What is the upload rate in p2p mode chrome may decide there is not enough bandwidth and will stop sending video and you will experience the same you see in the screenshots.

**Comment by venkatesh-jaam** [comment]
- In P2P, I checked the websocket section in developer console. In chrome, the upload rate was 0. Also the packet loss was 0.

```
video: {
    upload: 0,
    download: 1358
}
```

Whereas in Firefox I could see the video upload and download rate other than 0. This made me suspicious that if the video is even uploaded to the stream.

- I could see the changes when a video is stopped due to bandwidth. I did by adding one more participant from another chrome instance.

![received_251139005854170](https://user-images.githubusercontent.com/69686265/90864935-22104780-e381-11ea-8651-49f3baade17f.png)

Also, If I enter the meeting with video on, there is no issues.
So this issue is clearly not related Bandwidth.
